psalm.xml

```diff
@@ -147,6 +147,7 @@
                 <referencedMethod name="Symfony\Component\HttpKernel\Event\RequestEvent::isMasterRequest" />
                 <referencedMethod name="Symfony\Component\HttpKernel\Event\ExceptionEvent::isMasterRequest" />
                 <referencedMethod name="Symfony\Component\HttpKernel\Event\ResponseEvent::isMasterRequest" />
+                <referencedMethod name="Symfony\Component\Config\Definition\Builder\NodeDefinition::children" />
```

Comment:
Is this change related to gulp? Looks like should have been done at 1.13 branch.

Reply:
1.13 is working, idk why the build broke here. I'll check that again, I'll consider moving these two changes into a separate PR.

Reply:
Ok, I remember. This one has appear after removing the `->booleanNode('use_webpack')->defaultTrue()->end()` line in the configuration. I've checked my PR adding this line, and it's an obvious bug of Psalm (I don't like it tbh). So yes, we might consider this change as a part of removing Gulp 🤣.